avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -1591,20 +1594,28 @@ function Set-UsageExamplesSection {
                 }
             }
 
-            # [5/6] Convert Bicep parameter block to JSON parameter block to enable processing
+            # [4/6] Convert Bicep parameter block to JSON parameter block to enable processing
             $conversionInputObject = @{
                 BicepParamBlock = ($paramsBlockArray | Out-String).TrimEnd()
                 CurrentFilePath = $testFilePath
             }
             $paramsInJSONFormat = ConvertTo-FormattedJSONParameterObject @conversionInputObject
 
-            # [6/6] Convert JSON parameters back to Bicep and order & format them
+            # [5/6] Convert JSON parameters back to Bicep and order & format them
             $conversionInputObject = @{
                 JSONParameters         = $paramsInJSONFormat
                 RequiredParametersList = $RequiredParametersList
             }
             $bicepExample = ConvertTo-FormattedBicep @conversionInputObject
 
+            # [6/6] Convert the Bicep format to a Bicep parameters file format
+            $bicepParametersfileExample = $bicepExample -split '\r?\n' | ForEach-Object {
```

Comment:
Suggestion for all occurrences
```suggestion
            $bicepParameterFileExample = $bicepExample -split '\r?\n' | ForEach-Object {
```

Reply:
As discussed, I'll leave it as it is.

Reply:
OUp to you to leave it as per documentation. I'd anyway suggest to at least use camelCase
```suggestion
            $bicepParametersFileExample = $bicepExample -split '\r?\n' | ForEach-Object {
```

This applies also to other parameters. such as `formattedBicepParametersfileExample` and `addBicepParametersfile`